src/components/Palmid/Palmid.tsx

```diff
@@ -26,7 +26,17 @@ export const Palmid = () => {
     )
     const [isInfoCollapsed, setIsInfoCollapsed] = React.useState<boolean>(true)
     const [isFastaCollapsed, setIsFastaCollapsed] = React.useState<boolean>(true)
-
+    const search = window.location.search
+    const params = new URLSearchParams(search)
+    const input = params.get('fastaInput')
+    useEffect(() => {
+        if (input) {
+            setFastaInput(input)
+        }
+        return () => {
+            setFastaInput('')
+        }
```

Comment:
Is this needed since the default value is already `''`?

https://github.com/serratus-bio/serratus.io/blob/d039da168d77047f4c5f4a8fdf69293c87b2855f/src/components/Palmid/Palmid.tsx#L22

Reply:
Oh I see, this is the cleanup function. Made a slight adjustment in 4d8ee25 so it's more clear.